docs/README.md

```diff
@@ -48,37 +48,25 @@ Like most frameworks, luma.gl has been developed with a certain philosophy in mi
 
-The arrival of WebGL2 was a major milestone in the WebGL landscape. With the release of luma.gl v4 in July 2017, luma.gl was positioned as a foundation library for high-performance GPU programming in JavaScript, and luma.gl v5 and v6 series releases have continued to provide incremental improvements in the WebGL2 and GPGPU areas.
-
-Today, luma.gl is the foundational WebGL library in the vis.gl framework suite, but is designed so that it can be used stand-alone.
+Today, luma.gl is the core 3D rendering library in the vis.gl framework suite, but is designed so that it can be used on its own.
 
 
 ## Comparison with other WebGL frameworks
 
 luma.gl is the natural choice if you are working with any of the WebGL-based frameworks in the vis.gl suite (deck.gl, kepler.gl etc).
 
-If not, and you are considering what WebGL framework to use in an independent project, then as a first step towards a decision we recommend that you take a look at the things that have been built on luma.gl (e.g. vis.gl), make sure the luma.gl design philosophy resonates with you, and finally, review the luma.gl roadmap to see that the library is heading in a direction that is meaningful to you.
-
-We think that luma.gl could be a great choice if:
-
-* you want to work with, learn, and leverage the power of the WebGL2 API.
-* you want "more control" by having the option of working close to WebGL.
-* you want to do shader coding
-* you are focusing on rendering large data sets with high performance.
-
-We feel that luma.gl is currently not the strongest choice if:
-
-* You wish to avoid learning anything about WebGL, working exclusively with higher abstractions.
-* You need to load 3D models from various formats.
-* You need traditional game engine support for scenegraphs, complex materials, advanced lighting options, etc.
+Otherwise, luma.gl is a strong choice for the following requirements:
+ * Low-level access to WebGL 2 constructs: programs, shaders, buffers, etc.
+ * To program against the WebGL 2 API, with seamless fallbacks to WebGL 1 for functionality that can be polyfilled via, for example, extensions to the WebGL 1 API.
+ * A focus on drawing large number of objects with minimal overhead.
 
-If you are considering luma.gl because you are interested in using WebGL2 and you are looking for options, it is worth noting that some of the more mature WebGL frameworks have been rather slow to upgrade to WebGL2, either because of historical reasons (e.g. existing higher-level APIs do not provide an obvious upgrade path) or for philosophical reasons (e.g. maintainers feel that WebGL2 is not worthwhile for their particular use cases).
+Note, however that luma.gl is not a game engine or a scenegraph library, and as such, does not provide higher-level abstractions like a transform hierarchy, materials, animation systems.
```

Comment:
This is not completely correct as we have some of this stuff in v7.

Maybe soften somewhat

"luma.gl is not intended to be a state-of-the-art game engine or scenegraph library and provides limited support for ... "